portal/autoconfig.py

```diff
@@ -53,9 +53,22 @@
         'django_otp.plugins.otp_static',
         'django_otp.plugins.otp_totp',
     ],
+    'PIPELINE_COMPILERS': (
+        'pipeline.compilers.sass.SASSCompiler',
+    ),
+    'PIPELINE_CSS': {
+        'css': {
+            'source_filenames': (
+                'portal/sass/*.scss',
```

Comment:
can you investigate when it happens on the pipeline because of the hardcoded path :)